### ISSUE TITLE
Going from home page to studio, it take you to looks like general management of last created event.  Currently my organization does not have any events. 

Clicking return to studio takes you to studio showing no events(which I presume to be correct)

### ISSUE THREAD

**Issue by zmelamed** [big-neon/bn-web#521]
[https://www.bugherd.com/projects/154576/tasks/6](https://www.bugherd.com/projects/154576/tasks/6)

**Comment by Jasonvdb** [comment]
@zmelamed I'm not sure I understand this one. I did find that box office is populating the event dropdown selection with all the events and not just the current orgs events. Do you think that's what this is about?

**Comment by zmelamed** [comment]
Yes, this issue is around the the box office allowing you to see events that aren’t part of your org I believe
